administrator/templates/atum/scss/vendor/fontawesome-free/fontawesome.scss

```
@@ -0,0 +1,16 @@
+// Override the font path
+$fa-font-path: "../../../../../../../../media/vendor/fontawesome-free/webfonts" !default;
```

```suggestion
$fa-font-path: "../../../../../../media/vendor/fontawesome-free/webfonts" !default;
```
It is now working with this change.

---

Thanks